server/services/allocations.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable camelcase */
+import { HcapUserInfo } from '../keycloak';
```

Comment:
Is it worth disabling the camelcase rule across the entire server? We seem to be disabling this rule in several files.

Reply:
I think I prefer how it is at the moment. If anything, I'd argue that we should be working to refactor out as much camelcase as possible. One potential solution there would be a services-wide utility function that converts snake case to camel case (and an inverse function), which we'd then use on all DB operations to ensure all JS project-wide is seeing nothing but camels. A discussion for another time, though.